src/app/modules/submission/multirespondent-submission/multirespondent-submission.controller.ts

```diff
@@ -365,6 +475,7 @@ export const handleUpdateMultirespondentSubmission = [
   MultirespondentSubmissionMiddleware.createFormsgAndRetrieveForm,
   MultirespondentSubmissionMiddleware.scanAndRetrieveAttachments,
   // EncryptSubmissionMiddleware.validateStorageSubmission,
+  MultirespondentSubmissionMiddleware.setCurrentWorkflowStep,
```

Comment:
/nit For discussion, non-blocking

I recall that we have logs disabled for middlewares, i.e., makes debugging difficult